.env.template

```diff
@@ -1,7 +1,38 @@
+# AWS Secrets
+# These credentials are baked into the image to be used by the application
 export AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
 export AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
-export DEV_AWS_ACCESS_KEY_ID="tHiSiSyOuRdEvAcCeSsKeY"
-export DEV_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRdEvAcCeSsSeCrEt"
+
+# This is a non-standard name used in this repo;
+# it is tech debt and should be cleaned up to use the standard name
+export AWS_ACCESS_SECRET=$AWS_SECRET_ACCESS_KEY
+
+# These are only used by GitHub Actions, and use secrets of the same names
+# configured on GitHub; they are listed here only for the sake of completeness
+export DEV_AWS_ACCESS_KEY_ID=
+export DEV_AWS_SECRET_ACCESS_KEY=
+export STAGING_AWS_ACCESS_KEY_ID=
+export STAGING_AWS_SECRET_ACCESS_KEY=
+export PROD_AWS_ACCESS_KEY_ID=
+export PROD_AWS_SECRET_ACCESS_KEY=
+
+# These credentials are used by Packer to build the AMI
+# See the item "AWS Github/Terraform build access key" in Bitwarden
+export DEPLOY_AWS_ACCESS_KEY_ID="tHiSiSyOuRAcCeSsKeY"
+export DEPLOY_AWS_SECRET_ACCESS_KEY="tHiSiSyOuRAcCeSsSeCrEt"
```

Comment:
Okay, now that I'm actually trying to do this I'm confused again!  Should these be the same as "AWS_ACCESS_KEY_ID" and "AWS_SECRET_ACCESS_KEY" above?  Assuming not or else you would have indicated that.  So if I use the build access keys here, do I use my personal ones above?

Reply:
They can be the same; you can probably even leave the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` as the placeholders, and the app just won't be able to talk to AWS, which is fine for the purpose of testing "can I build an image". I don't actually know if personal credentials will lead to a working app!

The `DEPLOY` credentials must be valid, though, and must be able to create AMIs and EC2 instances and probably some other things, as packer uses these credentials directly.

Reply:
To follow up on what we discussed in slack: the credentials in `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` need to be able to read the `permanent-repos` S3 bucket, so placeholders will not work there. They can be the same as the `DEPLOY` credentials, though.

Reply:
Can we move the deploy keys up and then do:
```
export AWS_ACCESS_KEY_ID=$DEPLOY_AWS_ACCESS_KEY_ID
export AWS_SECRET_ACCESS_KEY=$DEPLOY_AWS_SECRET_ACCESS_KEY
```

?

Reply:
Hmm... I'm a little hesitant to suggest that in the documentation, because this is definitely a hacky thing that I did (and suggested you do) for the sake of expediency. The key the application uses should not be the same as the key we use to deploy, and honestly the key we use for manually testing image creation should not be the same as the key we use for automated image creation. What we'd ideally do is figure out what permissions the different keys need and document them, but I don't think that's the highest priority and I don't propose spending that time now.

But... I guess this is primarily intended for developers, and documentation of a bad process is better than no documentation of a bad process. I'll try to figure out a TODO to at least suggest that we shouldn't be doing this.

Reply:
Fixup commit https://github.com/PermanentOrg/infrastructure/pull/51/commits/97caae5c88f2cab49f68c80772750be8730fdf46 ; will squash and merge momentarily.